### ISSUE TITLE
Uncaught TypeError: Cannot read property 'setTarget' of undefined

### ISSUE THREAD

**Issue by admosity** [t9md/atom-vim-mode-plus#216]
[Enter steps to reproduce below:]

1. Hit `?` from normal mode

**Atom Version**: 1.6.1
**System**: Mac OS X 10.11.3
**Thrown From**: [vim-mode-plus](https://github.com/t9md/atom-vim-mode-plus) package, v0.29.0

### Stack Trace

Uncaught TypeError: Cannot read property 'setTarget' of undefined

```
At /Users/cain/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:53

TypeError: Cannot read property 'setTarget' of undefined
    at OperationStack.reduce (/Users/cain/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:119:21)
    at OperationStack.process (/Users/cain/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:64:8)
    at OperationStack.run (/Users/cain/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:46:8)
    at Function.Base.run (/Users/cain/.atom/packages/vim-mode-plus/lib/base.coffee:239:31)
    at atom-text-editor.<anonymous> (/Users/cain/.atom/packages/vim-mode-plus/lib/base.coffee:234:66)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:570:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:378:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:26.5.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:25.8.0 find-and-replace:focus-next (atom-text-editor.editor.mini.is-focused)
     -0:25.8.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:22.7.0 find-and-replace:focus-previous (atom-text-editor.editor.mini.is-focused)
     -0:22.7.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:22.5.0 find-and-replace:focus-previous (atom-text-editor.editor.mini.is-focused)
     -0:22.5.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:20 core:confirm (atom-text-editor.editor.mini.is-focused)
 12x -0:19.9.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:06.8.0 core:close (ol.results-view.list-tree.focusable-panel.has-collapsable-children)
  7x -0:06.8.0 blur (ol.results-view.list-tree.focusable-panel.has-collapsable-children)
     -0:02.2.0 vim-mode-plus:search-backwards (atom-text-editor.editor.vim-mode-plus.visual-mode.characterwise.is-focused)
     -0:02.1.0 blur (atom-text-editor.editor.vim-mode-plus.visual-mode.characterwise.is-focused)
     -0:01.1.0 vim-mode-plus:search-insert-wild-pattern (atom-text-editor.editor.vim-mode-plus-search.mini.backwards.is-focused)
     -0:00.2.0 core:confirm (atom-text-editor.editor.vim-mode-plus-search.mini.backwards.is-focused)
  2x -0:00.2.0 blur (atom-text-editor.editor.vim-mode-plus-search.mini.is-focused)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "themes": [
      "one-dark-ui",
      "monokai"
    ]
  },
  "vim-mode-plus": {
    "incrementalSearch": true,
    "useSmartcaseForSearch": true
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.5.1
atom-beautify, v0.28.28
atom-jade, v0.3.0
atom-ternjs, v0.13.2
cursor-history, v0.5.9
docblockr, v0.7.3
emmet, v2.4.3
file-icons, v1.6.20
language-babel, v2.16.0
language-diff, v0.4.0
linter, v1.11.4
linter-htmlhint, v1.1.1
linter-jscs, v3.4.10
minimap, v4.21.0
minimap-git-diff, v4.2.0
minimap-highlight-selected, v4.4.0
minimap-split-diff, v0.3.0
monokai, v0.18.0
pigments, v0.26.0
transpose-selection, v1.1.0
vim-mode-plus, v0.29.0

# Dev
No dev packages
```

**Comment by t9md** [comment]
I believe duplicate #185.
Change I made in vmp-v0.30.0 possibly fix this issue.
Will close this issue to **re-evaluate** issue is still happen after vmp-v0.30.0.

**Comment by t9md** [comment]
After re-think, and investigation, this issue should be fixed in v0.30.0.
So I removed **re-evaluation** label. it should be fixed.
